applications/luci-app-filemanager/Makefile

```diff
@@ -0,0 +1,15 @@
+# This is free software, licensed under the Apache License, Version 2.0 .
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI File Manager module
+LUCI_DEPENDS:=+luci-base
+
+PKG_LICENSE:=Apache-2.0
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
```

Comment:
Remove this. git semantic versioning is fine.

Reply:
fixed